scripts/safekill.sh

```diff
@@ -1,15 +1,32 @@
 #!/usr/bin/env bash
 set -e
 
+# non-interactive shell won't source in ~/.bashrc by default
+# use our own bash configuration
+source ~/.bashrc_tmux_safekill
```

Comment:
should we test whether this file exists first?

Reply:
Thanks. I must have missed it. Updated.